src/bundle/Resources/public/js/scripts/admin.trash.list.js

```diff
@@ -1,6 +1,33 @@
-(function(global, doc, eZ, React, ReactDOM, Translator) {
+(function (global, doc, eZ, React, ReactDOM, Translator) {
+    let getUsersTimeout;
+    const CLASS_SORTED_ASC = 'ez-table__sort-column--asc';
+    const CLASS_SORTED_DESC = 'ez-table__sort-column--desc';
+    const CLASS_VISIBLE_DATE_RANGE = 'ez-trash-search-form__range-wrapper--visible';
+    const sortedActiveField = doc.querySelector('#trash_search_sort_field').value;
+    const sortedActiveDirection = doc.querySelector('#trash_search_sort_direction').value;
+    const dateFields = doc.querySelectorAll('.ez-trash-search-form__range-select');
+    const trashedTypeInput = doc.querySelector('#trash_search_trashed');
+    const token = doc.querySelector('meta[name="CSRF-Token"]').content;
+    const siteaccess = doc.querySelector('meta[name="SiteAccess"]').content;
+    const formSearch = doc.querySelector('form[name="trash_search"]');
+    const sortField = doc.querySelector('#trash_search_sort_field');
+    const sortDirection = doc.querySelector('#trash_search_sort_direction');
+    const creatorInput = doc.querySelector('.ez-trash-search-form__item--creator .ez-trash-search-form__input');
+    const usersList = doc.querySelector('.ez-trash-search-form__item--creator .ez-trash-search-form__user-list');
+    const resetCreatorBtn = doc.querySelector('.ez-btn--reset-creator');
+    const searchCreatorInput = doc.querySelector('#trash_search_creator');
+    const sortableColumns = doc.querySelectorAll('.ez-table__sort-column');
     const btns = doc.querySelectorAll('.btn--open-udw');
     const udwContainer = doc.getElementById('react-udw');
+    const autoSendNodes = doc.querySelectorAll('.ez-trash-search-form__item--auto-send');
+    const errorMessage = Translator.trans(/*@Desc("Users list fetching error")*/ 'users_fetch_error', {}, 'messages');
```

Comment:
It should be more like:
`Cannot fetch users list`
ping @DominikaK 

And it shouldn't be in the `messages` domain
The key should be more like: `trash.user_list.error`